api/buffer/design.go

```diff
@@ -142,6 +142,7 @@ var _ = Service("buffer", func() {
 			Meta("openapi:tag:configuration")
 			Response(StatusOK)
 			ResourceLimitReachedError()
+			AlreadyExistsError()
```

Comment:
Unifikoval jsem errory pro `already exists`, `not found` a `resource limit reached`